src/unstract/sdk/utils/callback_manager.py

```diff
@@ -130,3 +131,22 @@ def get_tokenizer(
         except OSError as e:
             logger.warning(str(e))
             return fallback_tokenizer
+
+    @staticmethod
+    @deprecated("Deprecated method. Use the latest set_callback instead")
```

Comment:
@gaya3-zipstack `deprecated` library will throw warning that it is deprecated, we do not need to mention again.

In that case let's change message to `Use set_callback() instead.`.

Reply:
Done